bot/handlers/bot_message_handler.py

```diff
@@ -78,6 +80,13 @@ async def _reply_invalid_args_count(
     def get_action_name(self) -> str:
         return self.__class__.__name__
 
+    async def get_response(self, key: str, args: Optional[List[str]] = None) -> str:
+        return await get_response(
+            key=key,
+            handler_name=self.get_action_name() ,
+            args=args,
+        )
```

Comment:
```suggestion
        return await get_response(key=key, handler_name=self.get_action_name(), args=args)
```

lekka upierdliwosc ale mysle ze nie ma co rozbijac takich krotkich normalnych linijek